src/test/ui/lifetime-errors/ex3-both-anon-regions.stderr

```diff
@@ -0,0 +1,10 @@
+error[E0623]: lifetime mismatch
+  --> $DIR/ex3-both-anon-regions.rs:12:12
+   |
+11 | fn foo(x: &mut Vec<&u8>, y: &u8) {
+   |                    ---      --- these references must have the same lifetime
```

Comment:
Are you saying -- if some other tool were to hide one of these labels, would that be ok? It seems like it would be too bad, but I'm not sure what to do about it. I guess we could offer multiple labels?